helix-core/src/main/java/org/apache/helix/spectator/RoutingTableProvider.java

```diff
@@ -558,23 +558,30 @@ protected void refresh(Map<String, Map<String, Map<String, CurrentState>>> curre
 
   private void resetRoutingTableAndNotify(long startTime, RoutingTable newRoutingTable) {
     _routingTableRef.set(newRoutingTable);
-    logger.info("Refresh the RoutingTable for cluster {}, takes {} ms.",
-        (_helixManager != null ? _helixManager.getClusterName() : null),
+    String clusterName = _helixManager != null ? _helixManager.getClusterName() : null;
+    logger.info("Refresh the RoutingTable for cluster {}, takes {} ms.", clusterName,
```

Comment:
Refresh -> refreshed, takes -> took. Generally shouldn't change the logs but we're changing it here anyways..

Reply:
changed.